src/app/tabs/camera.cpp

```diff
@@ -22,9 +25,20 @@ CameraTab::CameraTab(QWidget *parent) : QWidget(parent)
     layout->addWidget(this->local_camera_widget());
     layout->addWidget(this->network_camera_widget());
 
-    connect(this, &CameraTab::disconnected, [layout]() { layout->setCurrentIndex(0); });
+    connect(this, &CameraTab::disconnected, [layout,this]() {
+        layout->setCurrentIndex(0);
+        if (this->config->get_cam_autoconnect()) {
```

Comment:
what are the changes it just continually tries reconnecting forever and ever?

like if the reconnection happens, and the camera just isnt available, will `UnloadedStatus` be set, thus causing the reconnection to happen again?

Reply:
Yes, it tries reconnecting every X seconds (6 by default) forever, or until you uncheck the "automatically connect" checkbox.

The most important feature here is the connect on startup. The reconnect will probably only happen if someone unplugs a camera (like USB) or there's a network glitch.